src/gravel/controllers/resources/inventory.py

```diff
@@ -18,28 +18,17 @@ class Inventory(Ticker):
     _latest: Optional[NodeInfoModel]
 
     def __init__(self):
-        self.is_ticking = False
+        super().__init__(
+            "inventory",
+            gstate.config.options.inventory_probe_interval
```

Comment:
```suggestion
            gstate.config.options.inventory.probe_interval
```

nit, but wondering if we should split the `probe_interval` options by class?